### PR TITLE
Deprecate duplicate in favor of generic function copy

### PR DESCRIPTION
**Pull request by musm**
close: https://github.com/JuliaInterop/MATLAB.jl/issues/66

### CODE REVIEW

**Comment by musm** [comment]
Merge after: https://github.com/JuliaInterop/MATLAB.jl/pull/60

**Comment by musm** [comment]
Thoughts? This is heading in the large refactor direction to be more Julian, so I think we should do it.

File: src/MATLAB.jl [hunk 1]
```diff
@@ -68,4 +68,6 @@ function jvariable(mx::MxArray, ty::Type{AbstractString})
     return jstring(mx)::String
 end
 
+Base.@deprecate duplicate(mx::MxArray) copy(mx::MxArray)
```
Comment (yuyichao) [inline src/MATLAB.jl]: I'm pretty sure `@deprecate` is exported.

**Comment by musm** [comment]
merge?

**Pull request merged**